conda_recipes/README.md

```diff
@@ -131,6 +148,15 @@ You can add more files, like a LICENSE.txt to document the license of the recipe
 The file `deadline-cloud.yaml` file provides metadata for how to submit the package build
 job to Deadline Cloud.
 
+#### The buildTool option
+
+You can select the default build tool between conda-build and rattler-build for the whole recipe
+by setting this option.
+
+```
+buildTool: conda-build
+```
+
```

Comment:
It might help to include a bit more info about rattler-build in one of these docs sections. For example:
- rattler-build is generally faster than conda-build
- rattler-build has a slightly different recipe format so recipes are not 1-1 compatible (though they are very similar)
- any limitations to rattler-build besides being linux only?

Reply:
I've added some more info here, thanks for the review!